app/src/main/java/net/bible/service/readingplan/ReadingPlanTextFileDao.kt

```diff
@@ -243,21 +249,30 @@ class ReadingPlanTextFileDao {
             try {
                 // check to see if a user has created his own reading plan with this name
                 val userReadingPlanFile = File(USER_READING_PLAN_FOLDER, filename)
-                val isUserPlan = userReadingPlanFile.exists()
+                val userReadingPlanModule = AndBibleAddons.providedReadingPlans[planCode]
+                val isUserPlan = userReadingPlanFile.exists() || userReadingPlanModule?.file?.exists() == true
 
                 inputStreamRaw = if (!isUserPlan) {
                     assetManager.open(READING_PLAN_FOLDER + File.separator + filename)
                 } else {
-                    FileInputStream(userReadingPlanFile)
+                    if (userReadingPlanModule?.file?.exists() == true)
+                        FileInputStream(userReadingPlanModule.file)
+                    else
+                        FileInputStream(userReadingPlanFile)
                 }
 
                 val byteArrayForReuse = ByteArrayOutputStream().apply { write(inputStreamRaw.readBytes()) }
                 properties.load(ByteArrayInputStream(byteArrayForReuse.toByteArray()))
                 properties.planCode = planCode
                 properties.numberOfPlanDays = getNumberOfPlanDays(properties)
-                properties.versification = getReadingPlanVersification(properties)
-                properties.isDateBasedPlan = properties["1"].toString().contains("^([a-z]|[A-Z]){3}-([0-9]{1,2});".toRegex())
-                getNameAndDescFromProperties(ByteArrayInputStream(byteArrayForReuse.toByteArray()), properties)
+                properties.versification = getReadingPlanVersification(properties, userReadingPlanModule?.book?.getProperty(VERSIFICATION))
```

Comment:
@tuomas2 do you know a better way to get versification from Addon Book (reading plan module in this case)?

Reply:
book.versification.name works for JSword's `Book` class

Reply:
Hmm, it seemed that `Book` did not have any `versification` property or method. Maybe I'll have to look some more

Reply:
Ah it's in SwordBook only